rust-sdk/whirlpool/src/tests/token_test.rs

```diff
@@ -0,0 +1,126 @@
+use super::utils::{rpc::RpcContext, token::*, token_extensions::*};
+use crate::NativeMintWrappingStrategy;
+use serial_test::serial;
+use solana_sdk::signer::Signer;
+use spl_token::native_mint;
+use spl_token_2022::{extension::ExtensionType, ID as TOKEN_2022_PROGRAM_ID};
+
+#[tokio::test]
```

Comment:
I think preference would be to have tests in the same file as the code itself (which is what we are doing for the other rust packages as well). One exception of course would be the e2e test since that doesn't really relate to a single file/function

Reply:
Ok update to put it into token.rs